### ISSUE TITLE
upgrades: private state rollover during testnet

### ISSUE THREAD

**Issue by erwanor** [penumbra-zone/penumbra#3081]
**Is your feature request related to a problem? Please describe.**
In #2917, we were able to exercise the MVP upgrade functionality and test both: 1/ the coordination mechanism that node runners use to signal their intent to perform an upgrade 2/ the operational implications of doing so if the node runner chooses to, sharing context along the way. 

Since then, we have added more functionality like checkpointing the genesis state, and performing full `InitChain` handshakes with CometBFT, completely pruning the consensus state and adhering to the recommended best-practice by the CometBFT team.

During the course of testnet 62, we should re-perform this test:
- [x] with new tooling that incorporate @conorsch's feedback:
    - change the `export` command flag to `--home`
    - change the `upgrade` command flag to `--upgrade-path`
    - improve the command description
    - improve logging to explain the cause of halt when an upgrade height is reached
    - add an explainer message when a node refuses to restart because it the full node is in statis
- [x] exercising a full checkpointed genesis and `InitChain` handshake
- [x] upgrading a full node
- [x] #3483
- [x] #3482
- [x] #3485
- [x] #3484

Post 64:
- [ ] pointing a 07 lightclient at the chain and making sure that it can validate across the upgrade boundary

**Comment by erwanor** [comment]
This is important but not urgent, and blocked on #3093 - targeting 63.

**Comment by conorsch** [comment]
Marking as blocked until #3093 is resolved; then we can pick this back up.

**Comment by conorsch** [comment]
> pointing a 07 lightclient at the chain and making sure that it can validate across the upgrade boundary

This is the single remaining item tracked in the OP. However, it's blocked on #3506, so we've broken it out into its own ticket: #3577. Closing this as resolved.
